### ISSUE TITLE
Defect: Minor - Teaser Cards - The text and link button text on Teaser Cards differes and font size and not bolded respectively

### ISSUE THREAD

**Issue by Andrei-Boiangiu** [hlxsites/vg-volvotrucks-us#146]
On the Teaser cards, the font size of the text is slightly larger and the link button text is not bolded

**Steps to reproduce:**
1. Open https://www.volvotrucks.us/ and scroll down to the Teaser cards block components
2. Open https://main--vg-volvotrucks-us--hlxsites.hlx.page/ and scroll down to the Teaser cards block components
3. Notice and compare the font size of the text on the Teaser cards
4. Notice and compare the text in the link button on the Teaser cards

**Expected result**
Font size for text should be slightly smaller and text in the link button should in Bold - see left side of the attached screenshot
<img width="1910" alt="image (200)" src="https://user-images.githubusercontent.com/127409602/227470258-6a1607ac-af0c-4603-9c13-2086b61df18f.png">

**Actual result**
Font size for text is sligthly larger and the text in the link button is not in bold - see right side of the attached screenshot

<img width="1910" alt="image (200)" src="https://user-images.githubusercontent.com/127409602/227470563-4bef1257-e83f-4254-8631-5eb251e5b92c.png">

**Comment by Buuhuu** [comment]
We tried to align the font size across different blocks on the page. The main content has a font size of 16px and only a few teasers have 13px. We chose 16px a default, but should check again with the stakeholders. 

The same applies for the secondary CTA. It is not bold in other cases. Again we should check with the stake holders if this is the correct style, but we should try not to have different styles for the same thing on the same page.
